packages/core/src/features/css-layer.ts

```diff
@@ -0,0 +1,323 @@
+            if (readyForName) {
+                const layers: valueParser.WordNode[] = [];
+                for (const name of value.split('.')) {
+                    // ToDo: handle name duplications
+                    const splittedLayer = { ...node, value: name };
+                    if (layers.length) {
+                        layers.push({ ...node, value: '.' });
+                    }
+                    layers.push(splittedLayer);
+                    namedNodeRefs[name] = splittedLayer;
+                    names.push(name);
+                    readyForName = false;
+                }
+                ast.splice(i, 1, ...layers);
+            }
+        } else if (type === 'function' && value === GLOBAL_FUNC && readyForName) {
+            const globalName = globalValueFromFunctionNode(node);
+            if (globalName) {
+                namedNodeRefs[globalName] = node;
+                names.push(globalName);
+                globals[globalName] = true;
+            } else if (globalName === '') {
+                report.warn(atRule, diagnostics.MISSING_LAYER_NAME_INSIDE_GLOBAL());
+            }
+            readyForName = false;
+        } else if (type === 'div' && value === ',') {
+            readyForName = true;
+            multiple = true;
+        } else if (type === 'comment') {
+            // doesn't change meaning: do nothing
+        } else {
+            readyForName = false;
+            const source = valueParser.stringify(node);
+            report.error(atRule, diagnostics.NOT_IDENT(source), { word: source });
+        }
+    }
+    return {
+        names,
+        globals,
+        multiple,
+        transformNames(mappedNames: Record<string, string>) {
+            for (const [srcName, targetName] of Object.entries(mappedNames)) {
+                const modifiedNode = namedNodeRefs[srcName];
+                if (modifiedNode.type === 'function') {
+                    // mutate to word - this is safe since this node is not exposed
+                    (modifiedNode as any).type = 'word';
+                }
+                modifiedNode.value = targetName;
+            }
+            return valueParser.stringify(ast);
+        },
+    };
+}
+
+function analyzeCSSImportLayer(context: FeatureContext, importAtRule: postcss.AtRule) {
+    const ast = valueParser(importAtRule.params).nodes;
+    for (let i = 0; i < ast.length; ++i) {
+        const node = ast[i];
+        const { type, value } = node;
+        if (type === 'function' && value === 'layer' && node.nodes.length) {
+            for (const nestedNode of node.nodes) {
+                if (nestedNode.type === 'word') {
+                    for (const name of nestedNode.value.split('.')) {
+                        addLayer({
+                            context,
+                            name,
+                            importName: name,
+                            ast: importAtRule,
+                            global: false,
```

Comment:
is it always `global: false` ?

Reply:
local symbols may contain `global` boolean according to their definition, while imported alias are always true - in any case global transformation is done with a deep resolved symbol.